src/main/java/graph/SequenceGraph.java

```diff
@@ -5,20 +5,18 @@
 import java.util.Map;
 
 /**
-<<<<<<< HEAD
  * Our own Graph Class.
-=======
  * This class is the SequenceGraph.
  * A Graph handling a Directed-Acyclic-Graph.
  * This is our own data structure we will use to draw the eventual graph.
->>>>>>> master
  */
 public class SequenceGraph   {
 
 
     private Integer size;
     private HashMap<Integer, SequenceNode> nodes;
     private ArrayList<Edge> edges;
+    private ArrayList<ArrayList<SequenceNode>> columns;
```

Comment:
The same arraylist was already made in graphdrawer before, now it is created in a more logical place. It shouldn't decrease performance.